Abstract Syntax and Semantics Document/index.html

```diff
@@ -397,25 +397,18 @@
     <section>
       <h3>Substream</h3>
     
-    <p>A <em>substream</em> (also known as window) <code>S'</code> of a stream <code>S</code> is a subsequence of <code>S</code>.</p>
-    
-    <h4>Time-bounded Substream</h4>
-    
-    <p>A <em>time-bounded substream</em> is defined by two time instances providing a lower bound <code>t_l</code> and an upper bound <code>t_u</code> where <code>t_l &lt;= t_u</code>. A timestamped graph <code>(g_i,p_i,t_i)</code> is in the time-bounded substream if and only if <code>t_l &lt;= t_i &lt;= t_u</code>.</p>
-    
-    <h4>Count-bounded Substream</h4>
-    
-    <p>A <em>count-bounded substream</em> is defined by a time instance <code>t</code> and an integer value <code>n</code> that represents the number of timestamped graphs to include in the count-bounded substream. The count-bounded substream consists of the <code>n</code> timestamped graphs at or before time instance <code>t</code>. That is, a timestamped graph <code>(g_i,p_i,t_i)</code> is in the count-bounded substream if and only if there are less than or equal to <code>n</code> timestamped graphs between it and the time instance <code>t</code>.</p>
-    
-    <p  class="note">A bounded substream maintains the timestamped graph contexts of the original stream.</p>
-    
-    <blockquote>
-      <p>See <a href="https://github.com/streamreasoning/RSP-QL/issues/11">Issue 11</a>.</p>
-    </blockquote>
+    <p>A <em>substream</em> <code>S'</code> of an RDF stream <code>S</code> is an RDF stream that is isomorphic to a subsequence of <code>S</code>.</p>
+      
+      <p class="note">There are several specializations of the substream relation which are useful in practice. These include the <em>filter</em> relation, where
+      stream elements are selected based on satisfaction of some criterion, and the <em>window</em> relation, where a temporally contiguous portion of the stream is selected.
+      </p>
+      
+      <p class="ednote">Insert definition of <em>window</em> here, using the definition of <em>convex substream</em> from 
```

Comment:
the new definitions in #42 overcome limitations of the previous ones, they look fine to me